tests/wpt/mozilla/tests/mozilla/structuredclone.html

```diff
@@ -0,0 +1,32 @@
+<!doctype html>
+<meta charset="utf-8">
+<title></title>
+<html>
+<head>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+</head>
+<body>
+<script>
+async_test(function(t) {
+  var blob = new Blob(['<a id="a"><b id="b">hey!</b></a>'], {type:"text/plain"});
```

Comment:
1. I don't know. I'm under the impression that what we're doing is Spidermonkey specific, so I thought those tests belonged in the Mozilla folder.
2. I couldn't find a WPT test specifically testing structured cloning of blobs. I found tests for  [structured cloning image data](https://github.com/servo/servo/blob/master/tests/wpt/web-platform-tests/workers/interfaces/DedicatedWorkerGlobalScope/postMessage/structured-clone-imagedata.html) and [basic JS types](https://github.com/servo/servo/blob/master/tests/wpt/web-platform-tests/workers/interfaces/DedicatedWorkerGlobalScope/postMessage/structured-clone-message.html). 
There is also one test that uses [`postMessage` with a blob](https://github.com/servo/servo/blob/master/tests/wpt/web-platform-tests/offscreen-canvas/the-offscreen-canvas/offscreencanvas.convert.to.blob.w.html#L34) in `offscreen-canvas`.